### PR TITLE
enable C++ for gpgme to create gpgmepp packages

### PR DESCRIPTION
**Pull request by librehat**

### CODE REVIEW

File: packages/gpgme/gpgmepp-static.subpackage.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+TERMUX_SUBPKG_DESCRIPTION="Static libraries for gpgmepp"
```
Comment (Grimler91) [inline packages/gpgme/gpgmepp-static.subpackage.sh:1]: If you specify that the main gpgmepp subpackage contains `lib/libgpgmepp.*` then gpgmepp-static should be created automatically (if the build script works), see https://github.com/termux/termux-packages/blob/master/scripts/build/termux_create_subpackages.sh#L3,L8

Comment (librehat) [reply packages/gpgme/gpgmepp-static.subpackage.sh:1]: Oh neat!

Comment (xeffyr) [reply packages/gpgme/gpgmepp-static.subpackage.sh]: No, packages will go to the `gpgme-static`, there no `gpgmepp-static`. It doesn't split subpackages recursively.
```
~/.../debs-aarch64-1ea0340898a15ebeba1476be26f6446531d6fcfa/debs $ ls gpgme*.deb
gpgme_1.15.1-1_aarch64.deb  gpgmepp_1.15.1-1_aarch64.deb  gpgme-static_1.15.1-1_aarch64.deb
```

Comment (librehat) [reply packages/gpgme/gpgmepp-static.subpackage.sh]: 🥲 I'll revert the last commit

**Pull request merged**